docs/source/install.rst

```diff
@@ -12,15 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using `conda <https://www.anaconda.com/download/>`_::
+You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
 
-    conda install dask
+*   ``conda install dask``               : Installs dask from the default channel (recommended)
+*   ``conda install -c conda-forge dask``: Installs dask from `conda-forge <https://conda-forge.github.io/>`_.
 
-This installs Dask and all common dependencies, including Pandas and NumPy.
+This installs Dask and **all** common dependencies, including Pandas and NumPy.
 
-Dask packages are maintained both on the default channel and on and
-`conda-forge <https://conda-forge.github.io/>`_.
+Optionally, you can obtain a no frills dask installation using:
 
+*   ``conda install dask-core``                : Installs dask-core from the default channel (recommended)
```

Comment:
How about "Installs only the dask library".  I suspect that referring to `dask-core` as a thing will confuse people a bit.

Reply:
Good spotting, that was just a quick copy-edit-paste I did from the previous lines. I reckon we won't even need the description to be honest, dask-core seems to be self-explanatory. There is already an emphasis that 'dask' is the whole suite and 'dask-core' is well, the core/bare minimum. I'll just stick with ``conda install dask-core`` (with no description).